modules/local/dada2_taxonomy.nf

```diff
@@ -67,7 +67,7 @@ process DADA2_TAXONOMY {
         Genus = tx\$tax.Genus,
         confidence = tx\$confidence,
         sequence = tx\$sequence,
- 	row.names = names(seq)
+     row.names = names(seq)
```

Comment:
I dont get it. In the visualization here it looks worse, [in the current dev branch it looks better](https://github.com/nf-core/ampliseq/blob/dev/modules/local/dada2_taxonomy.nf) than [in the PR here](https://github.com/erikrikarddaniel/ampliseq/blob/fix-tabs/modules/local/dada2_taxonomy.nf).

Reply:
You're right. I must have edited this with my standard 2 space indentation sometime ago, and that got weird when I now updated the file automatically. I fixed this particular line in 21637e2. Did you see any other mistakes?